src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoServlet.scala

```diff
@@ -69,7 +71,7 @@ class EasyAuthInfoServlet(app: EasyAuthInfoApp) extends ScalatraServlet
         Ok(pretty(render(json)))
       case Success(None) => NotFound(s"$uuid/$path does not exist")
       case Failure(HttpStatusException(message, HttpResponse(_, SERVICE_UNAVAILABLE_503, _))) => ServiceUnavailable(message)
-      case Failure(HttpStatusException(message, HttpResponse(_, NOT_FOUND_404, _))) if message.startsWith("Bag ") => NotFound(s"$uuid does not exist")
+      case Failure(HttpStatusException(message, HttpResponse(body: String, NOT_FOUND_404, _))) if isBagDoesNotExistErrorMessage(message, body) => NotFound(s"$uuid does not exist")
```

Comment:
don't disclose that it is the bag that doesn't exist!

```suggestion
      case Failure(HttpStatusException(message, HttpResponse(body: String, NOT_FOUND_404, _))) if isBagDoesNotExistErrorMessage(message, body) => NotFound(s"$uuid/$path does not exist")
```

btw. after this change, some tests will fail, but I guess you can fix them yourself...